manifests/params.pp

```diff
@@ -63,28 +68,31 @@
       }
     }
     'ubuntu': {
-      # TODO do we really want to pull in ${distro_codename}-updates by default?
       case $xfacts['lsbdistcodename'] {
         'precise': {
           $legacy_origin      = true
           $origins            = [
             '${distro_id}:${distro_codename}-security', #lint:ignore:single_quote_string_with_variables
-            #'${distro_id}:${distro_codename}-updates', #lint:ignore:single_quote_string_with_variables
           ]
 
         }
-        'trusty', 'utopic', 'vivid', 'wily': {
+        'trusty', 'wily': {
```

Comment:
@igalic Ubuntu utopic [1] and vivid [2] have reached EOL and are / will be removed from the regular mirrors and migrated to the archive repositories.

The deprecation warning that is also part of the pull request will hint the user to this fact.

Do you see valid use cases for still supporting utopic and vivid?

Raoul

[1] https://lists.ubuntu.com/archives/ubuntu-announce/2015-July/000198.html
[2] https://lists.ubuntu.com/archives/ubuntu-announce/2016-January/000203.html